### ISSUE TITLE
Create "About" pages for Rime (and other poetic categories)

### ISSUE THREAD

**Issue by craigsapp** [TassoInMusicProject/tasso-website#45]
About pages should be created for Rime.  Add "About" link underneath each poem which discusses the poem

* Attribution, bibliographic references and such

* Dedicatee

* possible date of composition

* Commentary about the poem

* Text and translation of poem

* information about poem -- characters of the stanza

* On about page have an abbreviated list of the sources which expands when clicked on.

**Comment by craigsapp** [comment]
A basic about page has been created. 

For Rime, there is an additional link underneath each poem title on the main page:

<img width="1023" alt="screen shot 2017-04-17 at 5 23 23 pm" src="https://cloud.githubusercontent.com/assets/3487289/25109597/aa1cde0c-2392-11e7-950b-78a67c5d6cc8.png">


Clicking on that link to bring you to a page containing the poem:

<img width="998" alt="screen shot 2017-04-17 at 5 24 37 pm" src="https://cloud.githubusercontent.com/assets/3487289/25109608/c349cb10-2392-11e7-81b2-2d596a437fb9.png">

More information should be added.  Probably first to spreadsheets, and then eventually it will reside in the ATON index file:

https://github.com/TassoInMusicProject/tasso-website/blob/gh-pages/data/indexes/rime-verses.aton

(Gerusalemme and Aminta (and Other) would have their own verses index files (none yet for these categories).

Also only the Rime have "About" links since there are no online index content for the other poetry.

For now I have called the page "poem", since "about" is already taken for the header, describing the website.:

http://www.tassomusic.org/about

**Comment by craigsapp** [comment]
Additional metadata for about pages should be entered on the spreadsheet:

https://docs.google.com/spreadsheets/d/1YcaAQc-mxFFDWyOsT7HOlwZkEqB4rPEuOzXCMTKprMc/edit#gid=1607759727

There are 301 entries for all poetic groups (rime, gerusalemme, aminta and other).

I added columns for ATTRIBUTION and DEDICATEE.

For DEDICATEE, there is also an initial page created last year at:

https://docs.google.com/spreadsheets/d/1YcaAQc-mxFFDWyOsT7HOlwZkEqB4rPEuOzXCMTKprMc/edit#gid=1972477529

This can either be deleted or can be used as a database for URLs or other information about the dedicatees (such as birth-death dates), etc.

You can add columns to the first spreadsheet link above as needed, such as DATE for the date of composition of the poem (and/or PUBDATE for the date first published).  These will be added to the respective *-verses.aton files for each poetic category.

COMMENTARY should be added instead directly to the ATON index file:

https://github.com/TassoInMusicProject/tasso-website/blob/gh-pages/data/indexes/rime-verses.aton

This would be done by going to the correct catalog number for the literary work, and then adding a field starting with `@COMMENTARY:` followed by any number of lines containing the commentary.

(as well with TRANSLATIONS which we will have to first discuss since the rime-settings has that information at the moment for reasons you may have to explain to me).

**Comment by emilianoricciardi** [comment]
Hi Craig,

if we want to use the existing sheet for dedicatees, we will have to distinguish between dedicatees of poems and dedicatees of musical sources. Perhaps better, we can keep them separate, listing poems dedicatees in the "About" documents and musical sources dedicatees in the RIMS spreadsheet.

Emiliano

**Comment by craigsapp** [comment]
I think that the current dedicatee spreadsheet should go away (unless we want to add more information about dedicatees such as a url to a biography or other information).

I added a column to the rime data spreadsheet for rime dedicatees:

https://docs.google.com/spreadsheets/d/1YcaAQc-mxFFDWyOsT7HOlwZkEqB4rPEuOzXCMTKprMc/edit#gid=1098021743

<img width="1300" alt="screen shot 2017-04-17 at 8 19 15 pm" src="https://cloud.githubusercontent.com/assets/3487289/25113327/58ea8a8e-23ab-11e7-8ddd-2a63d62b1efb.png">


Literary dedicatees will be entered on the spreadsheet:

https://docs.google.com/spreadsheets/d/1YcaAQc-mxFFDWyOsT7HOlwZkEqB4rPEuOzXCMTKprMc/edit#gid=1607759727

<img width="998" alt="screen shot 2017-04-17 at 8 21 13 pm" src="https://cloud.githubusercontent.com/assets/3487289/25113336/6fa6f80c-23ab-11e7-8e65-1a1ee7c4661f.png">

**Comment by emilianoricciardi** [comment]
Since individual settings seldom have dedications, I'd add the dedicatee
column to the musical source/RISM sheet instead (composers would write
dedications for the source as a whole)

**Comment by craigsapp** [comment]
Yes, putting the dedicatees for the settings in the rism spreadsheet is best since that will reduce the redundancy from multiple works from the same rism volume.
